samples/DataSourceWithProcessingAndExtensions/README.md

```diff
@@ -0,0 +1,23 @@
+# Introduction 
+The Performance Toolkit SDK allows for you to create AddIns to process your own data for viewers (like Windows Performance Analyzer (WPA)). This project is a sample AddIn that you can use as a reference for creating your own AddIn. 
+
+
+# Requirements
+1. Visual Studio 2019
+2. .NET Standard 2.0
+
+# Configuring NuGet
```

Comment:
Given that the packages are now on NuGet.org, there isn't any special configuration needed anymore.